pkg/middleware/auth.go

```diff
@@ -16,6 +16,7 @@ import (
 type AuthOptions struct {
 	ReqGrafanaAdmin bool
 	ReqSignedIn     bool
+	NoAnonynmous    bool
```

Comment:
Nit: For consistency and helps make it clearer. Or alternatively `ReqAnonymous` and then flip the logic.
```suggestion
	ReqNotAnonynmous    bool
```

Reply:
ah, good suggestion!